Symfony/CS/Tokenizer/Transformer/ArrayTypehint.php

```diff
@@ -43,4 +43,12 @@ public function getCustomTokenNames()
     {
         return array('CT_ARRAY_TYPEHINT');
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function registerCustomTokens()
+    {
+        define('CT_ARRAY_TYPEHINT', self::getNextGeneratedCustomTokenValue());
```

Comment:
Of course, there should not be a performance loss, and I had to check this, if this pr had a chance to be merged.
But it seems that this is not the case.. ;)